vod/media_set.h

```diff
@@ -146,6 +153,10 @@ typedef struct {
 
 	media_notification_t* notifications_head;
 
+	media_closed_captions_t* closed_captions;
+	media_closed_captions_t* closed_captions_end;
+	bool_t has_closed_captions;
```

Comment:
I also kept this parameter because taking it out did not allow for differentiation between the case that the user provides an empty list of closed captions vs the user doesn't include the closedCaptions parameter at all.